pkg/volume/projected/projected.go

```diff
@@ -354,6 +354,27 @@ func (s *projectedVolumeMounter) collectData(mounterArgs volume.MounterArgs) (ma
 				Mode:   mode,
 				FsUser: mounterArgs.FsUser,
 			}
+
+			// Add service ca to ensure parity with service account token secrets
+			// TODO(marun) - Avoid requesting the configmap for every pod
+			//             - Only add to projected volumes added by BoundServiceAccountTokenVolume
+			//             - Handle when a source with name "service-ca.crt" already exists
+			caConfigMap, err := s.plugin.getConfigMap("openshift-config-managed", "service-ca")
+			if err != nil {
+				errlist = append(errlist, err)
+				continue
+			}
+			caCert := caConfigMap.Data["ca-bundle.crt"]
```

Comment:
Compare https://github.com/openshift/kubernetes/blob/4bbda77e59725f85aef253d522a33d5761027b9b/cmd/kube-controller-manager/app/patch_satoken.go#L37-L44 , and the surrounding code. Primarily the inclusion of the “rootCA”, and also that the data is read from files and not the API.

(I have no idea whether the old behavior is correct, I just want the difference to be an intentional decision.)